backend/worker/worker_test.go

```diff
@@ -705,115 +709,3 @@ func TestGetActiveDuration(t *testing.T) {
 		})
 	}
 }
-
-func TestFullSnapshotValidation(t *testing.T) {
```

Comment:
could we replace this with a test that has an incremental snapshot before a full snapshot, and see that we can process the session while just dropping the data before the full snapshot?